### ISSUE TITLE
Compatibility with 1-2S FCs

### ISSUE THREAD

**Issue by DoctorMikeReddy** [jumper047/LiteSilverware#6]
My Meteor75 Lite (from the Advanced Racing Kit 2) uses this FC:
https://betafpv.com/products/lite-1-2s-brushless-flight-controller-silverware-firmware-2
and
https://rcsearch.info/-c23913
which is a 1-2S FC board, which appeared in later Meteor75 Lites. Is the firmware compatible with this version?

The official BetaFPV github https://github.com/BETAFPV/LiteSilverware does not mention this variant :-(

**Comment by jumper047** [comment]
Seems like I can do nothing until BetaFPV will release sources for that board (add support from scratch is too much above my skills).

**Comment by DoctorMikeReddy** [comment]
﻿One is released now!
https://github.com/wenzhicode/LiteSilver
It is a separate repo as the 1-2s uses a different CPU

**Comment by jumper047** [comment]
Yep, thank you. Their new firmware is quite different from silverware, so I cant just apply commit to the new repo or something like this, I need to adapt my patches to firmware. May be I'll buy their new FC to play with it.. I'll notify you if I'll have any success.

**Comment by DoctorMikeReddy** [comment]
I have one of the MM based 1-2s FC boards (bought from drone-parts-center.com, but think they are out of stock now, so the brown lite F4 board looks like it was discontinued quickly) and one of the new F4 1-2s AIOs that is beta flight based; this was their only option when my original lite fc died in warranty. Not sure if the boards are cross compatible, so it looks like SilverLite was a brief experiment for them, which they are stepping away from

**Comment by Diversionpfe** [comment]
Hi jumper, is your source code compatible with this FC from BetaFPV:
https://betafpv.com/collections/brushed-flight-controller/products/betafpv-lite-brushed-flight-controller
I have this FC and was looking for its source code. Can you please be so kind to confirm?
Thkx

**Comment by jumper047** [comment]
Hi, sadly it is incompatible. But you can find the link to their sources in the messages above.

**Comment by Diversionpfe** [comment]
Thkx for pointing me in the right direction.
It's a pity not being compatible. Keep up the good work!
